test/core/security/tls_security_connector_test.cc

```diff
@@ -737,6 +737,44 @@ TEST_F(TlsSecurityConnectorTest,
                             on_peer_checked);
 }
 
+TEST_F(TlsSecurityConnectorTest,
+       ChannelSecurityConnectorWithVerifiedRootCertSubjectSucceeds) {
+  auto* sync_verifier_ = new SyncExternalVerifier(true);
```

Comment:
It doesn't seem to be a good pattern to introduce dependencies in test/cpp for test/core. It will be more tedious but I'd create a separate external verifier test class in core itself.

Reply:
Oh, nvm. I misread it.

Reply:
This is a pattern used in other tests in this file that I've re-used, I don't believe I've introduced a new dependency? Can you clarify what you mean?